eml/components/courses/courseCard/CourseCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { View, Text, Image, Pressable, Dimensions } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import { useFonts, VarelaRound_400Regular } from '@expo-google-fonts/dev';
+import { AppLoading } from 'expo-app-loading';
+
+export default function CourseCard({ course, downloadState }) {
+    const navigation = useNavigation();
+    
+    let [fontsLoaded] = useFonts({
+        VarelaRound_400Regular,
+      });
+    
+      if (!fontsLoaded) {
+        return AppLoading;
```

Comment:
Do i understand it correctly, that you just return program into an idling animation if the font is not loading?

Reply:
This is not needed, it is solved with the new Text component.